examples/single_model_scripts/run_swag.py

```diff
@@ -467,10 +467,13 @@ def main():
                 if (step + 1) % args.gradient_accumulation_steps == 0:
                     if args.fp16:
```

Comment:
We don't need this seperate adjustement of learning rate for fp16 anymore with the schedulers